recipe/meta.yaml

```
@@ -37,7 +37,7 @@ requirements:
     - {{ cdt('libselinux-devel') }}      # [linux]
     - pkg-config
     - perl
-    - libtool
+    - libtool  # [unix]
     - meson
     - ninja
   host:
```

CI logs suggest to add `libiconv` to `host`:

```
WARNING (gst-plugins-bad,Library/lib/gstreamer-1.0/gstsmoothstreaming.dll): Needed DSO Library/bin/iconv.dll found in ['libiconv']
WARNING (gst-plugins-bad,Library/lib/gstreamer-1.0/gstsmoothstreaming.dll): .. but ['libiconv'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
 ```

```suggestion
  host:
    - libiconv
```